CHANGELOG.md

```diff
@@ -4,35 +4,45 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## Unreleased
+## [Unreleased]
+
+## [0.2.0]
 
 ### Added
 
-- (GH-83) Allow for workspace configuration overrides
+- [(GH-83)](https://github.com/puppetlabs/pdkgo/issues/83) 31fc89a Allow for workspace configuration overrides
+- [(GH-107](https://github.com/puppetlabs/pdkgo/issues/107) 73a1a80 Initialize zerolog via cobra.OnInitialize method
+
+### Fixed
+
+- [(GH-15)](https://github.com/puppetlabs/pdkgo/issues/15) 06d5252 Unset necessary env vars in pdkshell
+- [(GH-125)](https://github.com/puppetlabs/pdkgo/issues/125) eb044da Fail on errors, quote arguments
+- [(GH-125)](https://github.com/puppetlabs/pdkgo/issues/125) 1eb56e5 Fix `$ver` bug in download script
```

Comment:
I would remove the commit SHAs or make it consistent across the board (and hyperlinked)

```suggestion
- [(GH-83)](https://github.com/puppetlabs/pdkgo/issues/83) Allow for workspace configuration overrides
- [(GH-107](https://github.com/puppetlabs/pdkgo/issues/107) Initialize zerolog via cobra.OnInitialize method

### Fixed

- [(GH-15)](https://github.com/puppetlabs/pdkgo/issues/15) Unset necessary env vars in pdkshell
- [(GH-125)](https://github.com/puppetlabs/pdkgo/issues/125) Fail on errors, quote arguments
- [(GH-125)](https://github.com/puppetlabs/pdkgo/issues/125) Fix `$ver` bug in download script
```

Reply:
Commit shas are automatically hyperlinked when viewed on GitHub

Reply:
TIL...

Fair enough, I'll know for next time. Probably good to update all, though